utils/create-icons-map.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+const svgFolder = '../assets/icons/';
+const iconMapFile = '../src/iconsMap.json';
+const svgCleaner = new RegExp('("fill:.*;")|(fill=".*")|(id=\'.*\' )|(width=".*")|height=(".*")', 'g');
```

Comment:
Note that this regex is greedy:
```
'<svg width="10" height="20">Blah</svg>'.match(/width=".*"/g);
// => ["width="10" height="20""]
```

Use this instead:
```
'<svg width="10" height="20">Blah</svg>'.match(/width="[^"]*"/g);
// => ["width="10""]
```

Reply:
I agree this regex token seems better, though using * is always greedy.
Maybe we could simplify to ``/(height|width|fill)="[^"]*"|fill:[^;]*;/g``
I don't think ID is necessary since we need it for <use> tag